generators/pkg/generator/generate.go

```diff
@@ -162,14 +163,14 @@ func GeneratePipeline(github *GitHub) (*v1beta1.Pipeline, error) {
 							Name: "REPO_FULL_NAME",
 							Value: v1beta1.ArrayOrString{
 								Type:      v1beta1.ParamTypeString,
-								StringVal: u.Path,
+								StringVal: strings.TrimPrefix(u.Path, "/"),
 							},
 						},
 						{
 							Name: "SHA",
 							Value: v1beta1.ArrayOrString{
 								Type:      v1beta1.ParamTypeString,
-								StringVal: "$(params.gitrevision)",
+								StringVal: github.Spec.Revision,
```

Comment:
We leave a comment / TODO about why we need to do this (e.g. param substitution wasn't working properly with finally).